styles.css

```diff
@@ -113,79 +90,49 @@ body {
   color: aqua;
 }
 
-.Introduction {
-  background: url(Assets/header.png) no-repeat center, #fff;
-  margin: 0;
-  border: 0;
-  width: 100%;
-  height: 600px;
-  border-radius: 0 0 0 75px;
-}
-
 .sample-project {
   background-color: #fff;
-  height: 720px;
   margin: 100px 24px;
   border-radius: 16px;
+  padding: 10px;
   display: grid;
   gap: 1px;
-  grid-template-areas: 'header header header' 'description description description' 'footer footer footer';
+  grid-template-areas: 'header header' 'description description';
   grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
 }
 
-.item {
-  width: 100%;
-  margin: 5px 0 0 0;
-}
-
 .project-header {
   grid-area: header;
-  padding: 20px 20px 10px;
-  height: 100%;
+}
+
+.project-description {
+  grid-area: description;
 }
 
 .image-holder {
-  display: flex;
   margin: auto;
-  justify-content: center;
-  align-items: center;
   border-radius: 8px;
-  width: 100%;
-  height: 100%;
 }
 
 .header-img {
   width: 100%;
-  height: 90%;
-  padding: 15px;
-}
-
-.project-description {
-  grid-area: description;
-  padding: 10px;
-}
-
-.title-container {
-  width: 100%;
-  display: flex;
-  flex-direction: column;
-  gap: 3px;
+  object-fit: contain;
 }
 
 .description-title {
   color: #172b4d;
   padding: 0 0 0 12px;
   width: 100%;
   line-height: 44px;
-  font-weight: 700;
-  font-size: 32px;
+  font-weight: 800;
+  font-size: 40px;
+  font-style: normal;
+  margin: 24px 0 0 0;
 }
 
 .subtitle-container {
-  width: 100%;
   font-size: 13px;
   font-weight: 600;
-  line-height: 16px;
   margin-top: 0;
   border-top: 0;
 }
```

Comment:
- [ ] In your 'subtitle-items' class, please disable word-wrapping and use flexbox to align items vertically. That would look more professional.
![image](https://user-images.githubusercontent.com/91486791/193357235-1de8d90e-0d8f-4225-bc2f-70534e6eef44.png)

Reply:
@tugsem Thank you. Will work on the suggestions and fix them.